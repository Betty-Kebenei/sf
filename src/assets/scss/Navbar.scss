#navbar {
    box-sizing: border-box;
    display: flex;
    padding: 0 10px;
    width: 100%;
    box-shadow: 0 7px 10px -10px #525252;

    .navlinks {
        padding-top: 25px;
        position: relative;
        left: 25%;

        ul {
            list-style-type: none;
            margin: 0;
            padding: 0;
           
            li {
                display: inline;
                padding: 0 20px;
            }
        }
    }
}
@media (max-width: 960px) {
    #navbar {
        display: block;
        text-align: center;
    
        .navlinks {
            padding-bottom: 10px;
            position: relative;
            left: 0%;
        }
    }
}

@media (max-width: 480px) {
    #navbar {
        display: block;
        text-align: center;
    
        .navlinks {
            padding-bottom: 10px;
            position: relative;
            left: 0%;

            ul {
                li {
                    display: block;
                    padding: 5px 0px;
                }
            }
        }
    }
}