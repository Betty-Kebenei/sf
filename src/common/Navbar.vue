<template>
  <div id="navbar">
    <div class="logo">
      <img
        alt="SF logo"
        src="../assets/images/logo.png"
      >
    </div>
    <div class="navlinks">
      <ul>
        <li>Home</li>
        <li>Goals and Achievements</li>
        <li>Activities</li>
        <li>Projects</li>
        <li>About</li>
        <li>Contacts</li>
      </ul>
    </div>
  </div>
</template>;

<script>
export default {
  name: 'Navbar',
};
</script>

<style lang="scss" >
   @import '../assets/scss/Navbar.scss';
</style>
