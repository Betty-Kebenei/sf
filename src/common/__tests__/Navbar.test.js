import { shallowMount } from '@vue/test-utils';
import Navbar from '../Navbar.vue';

describe('Navbar', () => {
  const wrapper = shallowMount(Navbar);
  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
