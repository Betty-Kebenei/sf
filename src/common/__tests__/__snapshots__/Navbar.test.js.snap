// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  id="navbar"
>
  <div
    class="logo"
  >
    <img
      alt="SF logo"
      src="../assets/images/logo.png"
    />
  </div>
   
  <div
    class="navlinks"
  >
    <ul>
      <li>
        Home
      </li>
       
      <li>
        Goals and Achievements
      </li>
       
      <li>
        Activities
      </li>
       
      <li>
        Projects
      </li>
       
      <li>
        About
      </li>
       
      <li>
        Contacts
      </li>
    </ul>
  </div>
</div>
`;
